nixos/tests/dendrite.nix

```diff
@@ -0,0 +1,102 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+  let
+    homeserverUrl = "http://homeserver:8008";
+
+    private_key = pkgs.runCommand "matrix_key.pem" {
+      buildInputs = [pkgs.dendrite ];
+    } "generate-keys --private-key $out";
+  in
+  {
+    name = "dendrite";
+    meta = with pkgs.lib; {
+      maintainers = teams.matrix.members;
+    };
+
+    nodes = {
+      homeserver = { pkgs, ... }: {
+        specialisation.running.configuration = {
```

Comment:
This is the culprit as it will build the configuration `running` but not run it. With
```suggestion
```
it should work.
The option `specialisation` can be used for switching configuration within a test.

Reply:
Should be resolved now, thanks!